packages/server/src/db/migrations/20200821121419_init.ts

```diff
@@ -0,0 +1,86 @@
+import * as Knex from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema
+    .createTable("users", (table) => {
```

Comment:
It's better to declare a file called `models.ts` and declare the object, which will bind table names with related models.

Example:

```typescript
const models = {
   USERS: {
     table: 'users',
     model: UserModel,
   }
}

// somewhere in a migration:

knex.schema.createTable(models.USER.table, ...)....
```

After that, you can reference the constant, which will make refactoring easier.
Imagine the case, when you need to just rename the table. Without having strong table name reference it would be a pain.